Tests/SwiftParserTest/translated/OperatorDeclTests.swift

```diff
@@ -167,7 +167,8 @@ final class OperatorDeclTests: XCTestCase {
       prefix operator %%+
       """,
       diagnostics: [
-        DiagnosticSpec(message: "unexpected code before operator declaration")
+        DiagnosticSpec(message: "operator must be declared as 'prefix', 'postfix', or 'infix'"),
```

Comment:
Why do we get an error message that the operator needs to be declared as `prefix` here? It is, right?